.github/workflows/tests.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+name: Tests
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: [8.1, 8.2]
+        symfony-version: ["^5.2", "^6.3"]
```

Comment:
The symfony-version is not used while installing composer dependencies, so the test are always running with `symfony/framework-bundle (v6.3.4)`